docs/Migrating to version 2.0 guide.md

```diff
@@ -404,6 +424,42 @@ Now you have to use it like:
 <ui5-dialog state="Success"></ui5-dialog>
 ```
 
+- `isOpen` and `close` methods are no longer present. Now `open` property can be used instead.
```

Comment:
We should restructure the sentence in order to start it with capital letter:
Methods `isOpen` and `close` are no longer present. Now `open` property can be used instead.

Reply:
Done.